app/controllers/sms/users_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Sms::UsersController < Sms::BaseController
+  def update
+    user = User.find_by_phone(params["From"])
+    if (params["Body"].upcase[0..1] == "NO" || params["Body"].upcase[0..10] == "UNSUBSCRIBE") && user
```

Comment:
I think that works. I think we would need to change the default on user creation for `active` to false. Does that sound right?